src/helpers/utils.js

```diff
@@ -0,0 +1,65 @@
+import { WrongElementTypeException } from 'exceptions/element';
+
+/**
+ * Remove the trailing slash from an URL and return it
+ * @param {string} url
+ * @return {string} - The cleaned URL
+ */
+export function removeTrailingSlash(url) {
+  return url.replace(/\/*$/, '');
+}
+
+/**
+ * Return a real copy of an object.
+ * Object.assign and the spread operators can not be used as they do not replace children objects
+ * by copies of themselves.
+ *
+ * Eg: If you use Object.assign or the rest operator on that object: { items: [{ childrenItem: 1 }] }
+ * All the objects contained in the items array will be references to the first objects
+ * @param {object} object
+ * @return {object}
+ */
+export function realObjectCopy(object) {
+  return JSON.parse(JSON.stringify(object));
```

Comment:
You're making the strong assumption that the object and its sub elements will only contain only Object, Array, String, Numbers, Bool and null.

Aka it will break on Map, Set and many other thinks AFAIK, and on circular references.

I would suggest to put that in the doc, and maybe to call it something like `simpleObjectDeepCopy`. `deepCopy` convers what you call "real" copy I assume, and the "simple" stand for the need of a certain type of object.

Reply:
It won't breaks on `Map` and `Set`, but replaces them by simple objects. But still, you made a point.

If the module had to cover all that use cases I'll had to replace it by the lodash `cloneDeep` method but in the current case it is not required. I'll add some infos as you suggested.

Thanks for your input!  o/